src/utils/color.ts

```diff
@@ -23,6 +23,8 @@ export function getFluxNodeTypeColor(fluxNodeType: FluxNodeType) {
       return "#f7d0a1";
     case FluxNodeType.System:
       return "#C5E2F6";
+    case FluxNodeType.LabelUpdater:
+      return "#F6E05E";
```

Comment:
why does it need a color?

Reply:
Typescript was complaining. Once I remove it from `FluxNodeType` I'll get rid of this too